### PR TITLE
#162662503 A landing page for users

### PR DESCRIPTION
**Pull request by kdaglas**
#### What does this PR do?
Enables users logged in or not to view the articles and a beautiful landing site when they access the website.

#### Description of Task to be completed?
- create the components for the footer, navigation bar, welcome banner, and the articles 
- write unit tests and integration tests.
-  use the CSS framework to design the layout of the page.
- fetch all Articles for the user to view them
- add the author of the Articles
- add read time of each article
- create reducers and actions to handle the communication with redux

#### How should this be manually tested?
- Fetch and checkout into this branch with `git fetch origin ft-landing-page-162662503 && git checkout ft-landing-page-162662503`
- create a `.env` file and your `BASE_URL` to it
- Install dependencies and start the app with `yarn install && yarn start`

#### What are the relevant pivotal tracker stories?
[#162662503](https://www.pivotaltracker.com/story/show/162662503)

### CODE REVIEW

**Comment by MuhweziDeo** [comment]
@kdaglas wow, the landing page looks amazing, exactly like the mockups. Just add the tests and everything will be set

File: src/actions/fetchArticlesAction.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import ActionTypes from './ActionTypes';
+
+const fetchArticlesAction = () => dispatch => fetch('https://ah-backend-xmen-staging.herokuapp.com/api/articles/?limit=6')
```
Comment (RrNn) [inline src/actions/fetchArticlesAction.js]: @kdaglas, as you proceed to work on this PR, let's create a `.env` file and put in our base **URL**. That way we wont have to do `fetch('https://ah-back....')`, you get the basic idea. We shall instead append whatever we need to append on the base url, so for this case we would have fetch(`BASE_URL/articles/?limit=6`).

Comment (kdaglas) [reply src/actions/fetchArticlesAction.js]: ok @RrNn am on it

**Review by MuhweziDeo — DISMISSED** [review]
LGTM @kdaglas @RrNn

File: src/components/LandingPage/ArticleCard.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import parseDate from '../../utils/neededFiles';
+
+export class ArticleCard extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
```
Comment (MuhweziDeo) [inline src/components/LandingPage/ArticleCard.js]: why is this here @kdaglas

**Review by armstrongsouljah — DISMISSED** [review]
It renders without crashing. LGTM!

**Review by malfahad — DISMISSED** [review]
LGTM too @kdaglas

**Review by mariamiah — DISMISSED** [review]
LGTM

**Review by calvinpete — DISMISSED** [review]
LGTM

File: src/components/LandingPage/ArticleCard.js (hunk 2 above)
Comment (RrNn) [reply src/components/LandingPage/ArticleCard.js]: @kdaglas 👆  Currently we don't need that state.
